community-website/package.json

```diff
@@ -12,17 +12,16 @@
     "test": "yarn lint && yarn docs:build && git checkout docs && git clean -f docs"
   },
   "devDependencies": {
-    "algolia-aerial": "^1.2.12",
-    "babel-eslint": "8.0.3",
-    "doctoc": "1.3.0",
-    "eslint": "4.12.1",
-    "eslint-config-algolia": "13.0.1",
+    "algolia-aerial": "^1.3.4",
```

Comment:
We should also pin this dependencies.

Reply:
They will all be pinned by renovate (I added it yesterday)